doc/releasenotes/15_0_0_release_notes.md

```diff
@@ -5,6 +5,10 @@
 
 ### New command line flags and behavior
 
+#### vtctl GetSchema --table_schema_only
```

Comment:
can you make this `--table-schema-only` please? ❤️ (it'll make flag standardizing less work for us, which we're coming to next)